src/serac/physics/base_physics.hpp

```diff
@@ -161,16 +161,16 @@ class BasePhysics {
   }
 
   /**
-   * @brief Register an externally-constructed FiniteElementState object as the source of values for parameter `i`
+   * @brief Deep copy a parameter field into the internally-owned parameter used for simulations
    *
    * @param parameter_state the values to use for the specified parameter
    * @param parameter_index the index of the parameter
```

Comment:
is it best practice to have the doxygen comments in the same order as the arguments?

Reply:
Yes. I'll fix this.